### PR TITLE
Add README.md

### PR DESCRIPTION
**Pull request by nunoguedelha**
Add README with a quick description of the tool and instructions for installing and running.

### CODE REVIEW

**Comment by nunoguedelha** [comment]
Ciao @traversaro , while the telemetry data servers have been moved to this repo, the Open MCT static server is still in the Open MCT tutorial repo, I'm still moving it to this one. Will submit the PR very shortly. The README assumes it's already done.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,135 @@
...
+    - Install Mamba, create a new environment and install the YARP package:
+    ```
+    conda install mamba
+    conda create -n robotologyenv
+    conda activate robotologyenv
+    mamba install -c robotology yarp
+    ```
+    To read more about installing `robotology` package binaries refer to https://github.com/robotology/robotology-superbuild/blob/master/doc/conda-forge.md#binary-installation.
+2. Install **node.js** and **npm**: you can follow the official guides in the respective web pages but we recommend to install them through [NVM](). Currently, the latest **node.js** LTS version compatible with **YarpJS** `master` commit [be28630](https://github.com/robotology/yarp.js/commit/be2863022713ded2fa48909404b43e98b09eeda2) is the LTS: Argon release **node.js v4.2.2** (refer to https://github.com/robotology/yarp.js/issues/19). Meanwhile, the latest **node.js** LTS version compatible with **Open MCT** is latest LTS:Fermium version **node.js v14.17.0**. For this reason we install both releases.
```
Comment (traversaro) [inline README.md:61]: The NVM link is missing. Furthermore, the structure of the sentence is a bit misleading: it starts with the "follow the official guides in the respective web pages", but that not the option we recommend. Perhaps we can write: 
```suggestion
2. Install **node.js** and **npm**: we recommende to install them via  [NVM](). Currently, the latest **node.js** LTS version compatible with **YarpJS** `master` commit [be28630](https://github.com/robotology/yarp.js/commit/be2863022713ded2fa48909404b43e98b09eeda2) is the LTS: Argon release **node.js v4.2.2** (refer to https://github.com/robotology/yarp.js/issues/19). Meanwhile, the latest **node.js** LTS version compatible with **Open MCT** is latest LTS:Fermium version **node.js v14.17.0**. For this reason we install both releases.
```

Comment (nunoguedelha) [reply README.md:61]: Ok, I guess that "you can follow the official ..." can be assumed by default, so leaving just "we recommend" is more clear. There's a typo at "recommende".

Comment (nunoguedelha) [reply README.md]: Fixed in https://github.com/dic-iit/yarp-openmct/pull/2/commits/df23358e9bad17f9b87871dd76832d2948df6f72.

**Comment by nunoguedelha** [comment]
@traversaro , since the rework was minor, I assume it's ok. Merging.

**Pull request merged**